### DISCUSSION TITLE
Support for reactive apps today

### DISCUSSION THREAD

**Discussion by SharadKumar** [blitz-js/blitz#771] [General] [answered]
I am looking around and was reading through the Blitz documentation, releases, and issues. I am trying to assess if Blitz does, or intends to support reactivity in the apps?

I noted that [Prisma (v2) does not support subscriptions or realtime](https://github.com/prisma/prisma/issues/298) yet. The likes of Hasura has [decent support for it](https://hasura.io/blog/tagged/realtime/), and SWR too.

> With SWR, components get a stream of data updates constantly and automatically. Thus, the UI is always fast and reactive.

## Question
What would a workaround look like with Blitz (to have realtime UI) in the interim, should we decide to use Blitz? 
(we are quite excited to see the new [Auth release](https://blitzjs-com-git-auth.blitzjs.vercel.app/docs/auth) coming out soon!)

https://github.com/blitz-js/blitz/issues/586

**Comment by LoriKarikari** [comment, marked as answer]
Hi @SharadKumar, `react-query` which is what `blitz` uses, supports reactive queries.

**Reply by flybayer** [reply]
Yep, same as `swr`!

**Reply by SharadKumar** [reply]
Sounds good. Thanks for the clarification.
I think we need this explicitly called out in documentation somewhere.

**Reply by flybayer** [reply]
@SharadKumar good idea! Want to open a PR for that? 👉 https://github.com/blitz-js/blitzjs.com

**Reply by edi** [reply]
I just heard about Blitz in a medium daily sub email. Decided to glance through its features and its issues (while waking up, in bed lol).

While I have no experience with Rails or Ruby, I am interested in an active and flexible platform like Blitz. First thing I saw was the folder structures idea, which seems to be coming to fruition quite fast. So thats a plus.

Next, I obviously looked into realtime data streams, since I am an avid Firebase (now Firestore) user.

So I’ll give Blitz a try! If it does what it promises, damn you’re on the right track :)
